python-type-hints-multiple-types/01_type_hints_for_one_piece_of_data.py

```diff
@@ -0,0 +1,15 @@
+from typing import Union
+
+
+def parse_email(email_address: str) -> str | None:
+    if "@" in email_address:
+        username, domain = email_address.split("@")
+        return username
+    return None
+
+
+def parse_email(email_address: str) -> Union[str, None]:
```

Comment:
The linter complains about redefining a function. Maybe you can give the function a different name or put it in a comment like so:

```python
# def parse_email(email_address: str) -> Union[str, None]:
#     if "@" in email_address:
#         username, domain = email_address.split("@")
#         return username
#     return None
```